src/foremast/consts.py

```diff
@@ -155,7 +156,8 @@ def find_config():
                                  default='').split(','))
 GITLAB_TOKEN = validate_key_values(config, 'credentials', 'gitlab_token')
 SLACK_TOKEN = validate_key_values(config, 'credentials', 'slack_token')
-
+DEFAULT_TASK_TIMEOUT = validate_key_values(config, 'task_timeouts', 'default', default=120)
+TASK_TIMEOUTS = json.loads(validate_key_values(config, 'task_timeouts', 'envs', default="{}"))
```

Comment:
Something seems strange around this.
Should default be `default={}` and not a quoted {}? Also, why is this wrapped in a `json.loads()`?